requirements/common.in

```diff
@@ -33,7 +33,7 @@ python-jose[pycryptodome]  # from jose import jwt
 
 furl==2.1.0  # Imported as furl
 first  # Imported as first
-
+json-e  # import jsone
```

Comment:
Add json-e to the list (see https://github.com/mozilla/treeherder/commit/3f73a3103ec8fe4b332d388c8db2d562302b6a80#diff-b80ca39e5ba09203c741fcc8e103bd16R515